Add missing tools

jsf9k opened the pull request:
## 🗣 Description ##

This pull request adds some tools to the Kali AMI that were requested by the users.

## 💭 Motivation and context ##

Resolves #137.

## 🧪 Testing ##

All automated tests pass.  I also created a staging AMI with these changes and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

jsf9k committed:
Bump version from 0.5.12 to 0.5.13

jsf9k committed:
Bump version from 0.5.13 to 0.5.13-rc.1

jsf9k committed:
Add several packages

These were requested by the assessors in #137.

jsf9k committed:
Add two tools to be installed from source

These tools were requested by the assessors in #137.

jsf9k committed:
Install golang package

This was requested in #137.

jsf9k committed:
Update comment

I don't believe the previous content is still true; Python 2 may get
reinstalled, but that no longer confuses Ansible.

dav3r reviewed (approved):
Looks great - thanks for taking care of this!

jmorrowomni commented:
lgtm

jsf9k committed:
Add GOPATH binary paths to PATH for the VNC user

This is a little complicated because GOPATH can contain a single path
or multiple paths separated by colon delimiters, and we need to
consider each case separately.

This was requested in #137.

jsf9k committed:
Bump version from 0.5.13-rc.1 to 0.5.13-rc.2

mcdonnnj commented on src/assessor_tools.yml:
Unless I've misunderstood something.

```suggestion
        # The jinja2 comment below changes the Jinja2 templating to
```

mcdonnnj reviewed (approved):
Looks about right to me. One suggestion about a comment for you but otherwise ✔

jsf9k committed:
Finalize version from 0.5.13-rc.2 to 0.5.13
